defaults/main.yml

```diff
@@ -1,2 +1,3 @@
 ---
 # defaults file for ansible-aws-cloudwatch-logs-agent
+stream_name: "{{ ansible_ec2_instance_id }}"
```

Comment:
I think it may be best to leave the default as `{instance_id}`. Specifically, cases where tools like Packer or Animator are used with Ansible provisioners to create Amazon Machine Images. Otherwise, the instance id of the machine used to make the image would be used by instances launched from that AMI.